From 60dc3a43ceeac2fb9cc10b54dd847f9425908390 Mon Sep 17 00:00:00 2001
From: Colin Ian King <colin.king@canonical.com>
Date: Tue, 22 Jun 2021 16:17:42 +0000
Subject: [PATCH 27/35] block/mq-deadline: remove redundant assignment of
 variable ret

Variable ret is being assigned a value at the end of the function
and the value is never read. The assignment is redundant and can
be removed.

Signed-off-by: Colin Ian King <colin.king@canonical.com>
---
 block/mq-deadline-main.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/block/mq-deadline-main.c b/block/mq-deadline-main.c
index cdfed612c..9874c8ec4 100644
--- a/block/mq-deadline-main.c
+++ b/block/mq-deadline-main.c
@@ -639,7 +639,6 @@ static int dd_init_sched(struct request_queue *q, struct elevator_type *e)
 	if (ret)
 		goto free_stats;
 
-	ret = 0;
 	q->elevator = eq;
 	return 0;
 
-- 
2.33.0.rc0.52.ge5a14ddd2d

